src/main/java/io/seqera/tower/cli/commands/LaunchCmd.java

```diff
@@ -165,7 +168,8 @@ private WorkflowLaunchRequest updateLaunchRequest(WorkflowLaunchRequest base) th
                 .optimizationTargets(coalesce(adv().disableOptimization, false) ? null : base.getOptimizationTargets())
                 .labelIds(base.getLabelIds())
                 .headJobCpus(base.getHeadJobCpus())
-                .headJobMemoryMb(base.getHeadJobMemoryMb());
+                .headJobMemoryMb(base.getHeadJobMemoryMb()
+                .launchContainer(coalesce(launchContainer, base.getLaunchContainer())));
```

Comment:
From where `base.getLaunchContainer()` come from ?

Reply:
base is WorkflowLaunchRequest
https://github.com/seqeralabs/tower-cli/blob/dafe28cf29831c0a51bb67dfc9551501e1c3d3b0/src/main/java/io/seqera/tower/cli/commands/LaunchCmd.java#L146
its here
https://github.com/seqeralabs/platform/blob/06e506f2eab2ade11dbc5d46d5cc1d19185b2170/tower-enterprise/src/main/groovy/io/seqera/tower/exchange/workflow/WorkflowLaunchRequest.groovy#L63

Reply:
but it needs a new release of SDK to be there in WorkflowLaunchRequest

Reply:
Don't think it can come from the `base` request. This was never supported before, therefore i'd expect it can only be provided as CLI option.